express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const grades = [];
+let nextId = 1;
+
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  res.status(201);
+  req.body['id:'] = `${nextId}`;
```

Comment:
you can just use `.id` and leave the `nextId` as a number

Reply:
Got it! Updated